includes/class-wp-job-manager-email-notifications.php

```diff
@@ -99,6 +99,39 @@ public static function send_deferred_notifications() {
 		}
 	}
 
+	/**
+	 * Send email notifications for a specific email now and return the status. Not usually necessary but useful if you
+	 * need to check to make sure a notification was sent.
+	 *
+	 * @param string $email_key Email key to send notifications for.
+	 * @return bool True if an email notification was sent.
+	 */
+	public static function send_notifications( $email_key ) {
+		$email_sent = false;
+		$email_notifications = self::get_email_notifications( true );
+		foreach ( self::$deferred_notifications as $index => $email ) {
+			if (
+				! is_string( $email[0] )
+				|| ! isset( $email_notifications[ $email[0] ] )
+				|| $email[0] !== $email_key
+			) {
+				continue;
+			}
+
+			$email_class            = $email_notifications[ $email[0] ];
+			$email_notification_key = $email[0];
```

Comment:
Isn't this just `$email_key`?

Reply:
Yup! 7e0172da42a92c2aa3d5bc60adbc8634124b94fd